### ISSUE TITLE
FEAT: 

### ISSUE THREAD

**Issue by jeanvanwyklf** [beekeeper-studio/beekeeper-studio#731]
**Is your feature request related to a problem? Please describe.**
AND / OR parts on a JOIN is not very visible

**Describe the solution you'd like**
It would be useful if the AND / OR statements on a JOIN is indented one more level
Please format the **Formatted SQL** similar to the SQL in the **Paste SQL** (JOIN AND + OR) in the attached screenshot

**Describe alternatives you've considered**
Make no change.

**Additional context**

![SQLTools-format](https://user-images.githubusercontent.com/86206824/126240534-f3b9b0d9-8cca-4579-bcc2-2a0f157440af.png)

**Comment by rathboma** [comment]
Yes! I agree with you. We're using a sql formatting library, so this will require a PR to that, but I agree that isn't formatted right.
